backend/src/api/categories/categories.api.ts

```diff
@@ -27,6 +27,17 @@ const initCategoriesApi: FastifyPluginAsync<Options> = async (
     },
   });
 
+  fastify.route({
```

Comment:
```suggestion
```
`/courses` all (for the course management page)
`/courses/dashboard` (for the dashboard page)
all changes should be in the `courses` api